packages/headless/src/api/service/insight/get-interface/get-interface-config-response.ts

```diff
@@ -11,6 +11,7 @@ export interface InsightInterface {
   facets: Facet[];
   tabs: Tab[];
   settings: SettingsSection;
+  answerConfigId: string;
```

Comment:
Will there always be a string `answerConfigId` in the response, or can it be null or undefined?

Reply:
It can be null or undefined  indeed, @SimonMilord  this will need to be updated to be optional.